docker/tap-mssql.Dockerfile

```diff
@@ -1,6 +1,6 @@
 # docker build -f tap-mssql.Dockerfile -t dataopstk/tapdance:tap-mssql .
 
-FROM dataopstk/tapdance:tap-mssql-raw as tap
+FROM docker.pkg.github.com/aaronsteers/tap-mssql/tap-mssql:latest as tap
```

Comment:
You will want to change this to reference the slalom/tap-mssql repo now that it ahs been transfered

Reply:
Updated ✅